v3/src/components/graph/components/graph.scss

```diff
@@ -111,6 +111,7 @@
 
   &.active {
     border: 5px solid rgb(255, 216, 102);
+    z-index: 10;
```

Comment:
At some point we need to stop playing these ad hoc z-index games, but I guess now is not the time.